lib/vanagon/component.rb

```diff
@@ -215,7 +217,7 @@ def get_source(workdir) # rubocop:disable Metrics/AbcSize
         @dirname = './'
 
         # If there is no source, there is nothing to do to extract
-        @extract_with = ': no source, so nothing to extract'
+        extract_with << ': no source, so nothing to extract'
```

Comment:
FYI I'm not ditching this in this PR. It'd require strenuous testing due to how Makefiles are rendered (a lot of `&&`ing).